Sources/GRPCCodeGen/Internal/Translator/SpecializedTranslator.swift

```diff
@@ -19,7 +19,7 @@
 protocol SpecializedTranslator {
 
   /// The ``SourceGenerator.Configuration.AccessLevel`` object used to represent the visibility level used in the generated code.
```

Comment:
Docs are outdated here. Would you mind trying to build docs to see which other references may have broken?

Reply:
Yes, good catch. We should add a docc CI check as well.

Reply:
Ah, these were missed because DocC by default only checks public symbols. You can change that to internal if you use the docc-plugin (e.g. `swift package generate-documentation --target GRPCCore --symbol-graph-minimum-access-level internal`).

This surfaced a fair few issues with internal docs. I fixed a few, and ones introduced by this change, but I think we should tackle the rest separately.